src/components/sidebar/Menu.css

```diff
@@ -0,0 +1,30 @@
+.menu {
+  position: fixed;
+  background-color: var(--background-secondary-color);
+  width: 260px;
```

Comment:
avoid using fixed size use rems for responsiveness

Reply:
thank you , did the changes .